### ISSUE TITLE
Escaping a forward slash

### ISSUE THREAD

**Issue by Moomink** [milgra/sov#17]
If the window title contains "/", it will be displayed escaped.
These are present at the time of ``swaymsg -t get_tree``, but is there any plan to solve this problem with sov?
Thank you.

**Comment by milgra** [comment]
Hi, I will look into it, haven't had a title with slash so far.

**Comment by Moomink** [comment]
Thanks, it happens when I open a site with "/" in the title, like a Github repository. 

![2022-06-20-14_59_31](https://user-images.githubusercontent.com/52168914/174536969-848bf8ab-d92c-45d0-a75e-a67528c0e770.jpg)


Here is the output of swaymsg at that time.
```json
{
    "id":133,

"~~~~~some output~~~~~~~~":"",

    "name": "milgra/sov: An overlay that shows schemas for all workspaces to make navigation in sway easier. - Vivaldi",
    "window": 6291706,
    "nodes": [],
    "floating_nodes": [],
    "focus": [],
    "fullscreen_mode": 0,
    "sticky": false,
    "pid": 6533,
    "app_id": null,
    "visible": false,
    "max_render_time": 0,
    "shell": "xwayland",
    "inhibit_idle": false,
    "idle_inhibitors": {
        "user": "fullscreen",
        "application": "none"
    },
    "window_properties": {
        "class": "Vivaldi-stable",
        "instance": "vivaldi-stable",
        "title": "milgra/sov: An overlay that shows schemas for all workspaces to make navigation in sway easier. - Vivaldi",
        "transient_for": null,
        "window_role": "browser",
        "window_type": "normal"
    }
}
```

**Comment by milgra** [comment]
Fixed in [0.72](https://github.com/milgra/sov/releases/tag/0.72)
